src/main/java/frc/trigon/robot/subsystems/swerve/SwerveConstants.java

```diff
@@ -1,41 +1,45 @@
 package frc.trigon.robot.subsystems.swerve;
 
 import com.ctre.phoenix6.configs.Pigeon2Configuration;
-import com.pathplanner.lib.util.HolonomicPathFollowerConfig;
-import com.pathplanner.lib.util.PIDConstants;
-import com.pathplanner.lib.util.ReplanningConfig;
+import com.pathplanner.lib.config.PIDConstants;
+import com.pathplanner.lib.config.RobotConfig;
+import com.pathplanner.lib.controllers.PathFollowingController;
+import com.pathplanner.lib.trajectory.PathPlannerTrajectoryState;
 import edu.wpi.first.math.controller.PIDController;
 import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
 import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
 import edu.wpi.first.math.trajectory.TrapezoidProfile;
 import frc.trigon.robot.RobotContainer;
 import frc.trigon.robot.constants.RobotConstants;
 import frc.trigon.robot.poseestimation.poseestimator.PoseEstimatorConstants;
+import org.json.simple.parser.ParseException;
 import org.trigon.hardware.RobotHardwareStats;
 import org.trigon.hardware.phoenix6.pigeon2.Pigeon2Gyro;
 import org.trigon.hardware.phoenix6.pigeon2.Pigeon2Signal;
-import org.trigon.utilities.Conversions;
 
+import java.io.IOException;
 import java.util.function.DoubleSupplier;
 
-public abstract class SwerveConstants {
+public class SwerveConstants {
     private static final int PIGEON_ID = 0;
     static final Pigeon2Gyro GYRO = new Pigeon2Gyro(SwerveConstants.PIGEON_ID, "SwerveGyro", RobotConstants.CANIVORE_NAME);
     private static final double
             GYRO_MOUNT_POSITION_YAW = 0,
             GYRO_MOUNT_POSITION_PITCH = 0,
             GYRO_MOUNT_POSITION_ROLL = 0;
     private static final double
-            FRONT_LEFT_STEER_ENCODER_OFFSET = -Conversions.degreesToRotations(225.263672 - 360),
-            FRONT_RIGHT_STEER_ENCODER_OFFSET = -Conversions.degreesToRotations(-256.904297 + 360),
-            REAR_LEFT_STEER_ENCODER_OFFSET = -Conversions.degreesToRotations(108.369141),
-            REAR_RIGHT_STEER_ENCODER_OFFSET = -Conversions.degreesToRotations(-36.035156);
+            FRONT_LEFT_STEER_ENCODER_OFFSET = 0,
+            FRONT_RIGHT_STEER_ENCODER_OFFSET = 0,
+            REAR_LEFT_STEER_ENCODER_OFFSET = 0,
+            REAR_RIGHT_STEER_ENCODER_OFFSET = 0;
     private static final int
-            FRONT_LEFT_ID = 0,
-            FRONT_RIGHT_ID = 1,
-            REAR_LEFT_ID = 2,
-            REAR_RIGHT_ID = 3;
+            FRONT_LEFT_ID = 1,
+            FRONT_RIGHT_ID = 2,
+            REAR_LEFT_ID = 3,
+            REAR_RIGHT_ID = 4;
```

Comment:
why?

Reply:
I copied over the code from Killshon. I guess this is how it is now